x-pack/plugins/ml/common/constants/settings.ts

```diff
@@ -5,3 +5,5 @@
  */
 
 export const FILE_DATA_VISUALIZER_MAX_FILE_SIZE = 'ml:fileDataVisualizerMaxFileSize';
+export const ANOMALY_DETECTION_ENABLE_TIME_RANGE = 'ml:anomalyDetectionDefaultTimeRangeEnable';
```

Comment:
If you accept my suggestion for renaming the other setting, this one should be updated to match:
```suggestion
export const ANOMALY_DETECTION_ENABLE_TIME_RANGE = 'ml:anomalyDetectionTimeDefaultsEnable';
```

Reply:
Updated here ... although now the order is changed where the toggle to set it comes after the json field. Not totally ideal but I'm not sure what's a good solution around making the names in alphabetical order. 

![2020-09-01 at 2 43 PM](https://user-images.githubusercontent.com/43350163/91907088-d6a64500-ec6e-11ea-936c-7103d6923205.gif)